### ISSUE TITLE
Update eslint-plugin-json

### ISSUE THREAD

**Issue by Empact** [LN-Zap/zap-desktop#589]
It's currently the source of a security warning reported by `npm audit`. It hasn't been updated in 3 years but they're discussing the possibility here: https://github.com/azeemba/eslint-plugin-json/pull/18

```
┌───────────────┬──────────────────────────────────────────────────────────────┐
│ Low           │ Prototype Pollution                                          │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Package       │ lodash                                                       │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Patched in    │ >=4.17.5                                                     │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Dependency of │ eslint-plugin-json [dev]                                     │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Path          │ eslint-plugin-json > jshint > lodash                         │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ More info     │ https://nodesecurity.io/advisories/577                       │
└───────────────┴──────────────────────────────────────────────────────────────┘
```

Note this is a `devDependency` so not a security concern wrt the installed application.

**Comment by Empact** [comment]
Pending release of `jshint`

**Comment by simlu** [comment]
Relevant issue on jshint here: https://github.com/jshint/jshint/issues/3257

**Comment by jugglinmike** [comment]
Good news! JSHint released the patch for this today in version 2.9.6 http://jshint.com/blog/2018-07-30/release-2-9-6/
